docs/api/clipboard.md

```diff
@@ -4,6 +4,9 @@
 
 Process: [Main](../glossary.md#main-process), [Renderer](../glossary.md#renderer-process)
 
+In the renderer process context it depends on the [`remote`](remote.md) module on Linux,
```

Comment:
"In renderer process's it depends on..."

Reply:
Can we also raise a ticket to implement this with dedicated IPC or something so this isn't an issue

Reply:
bump

Reply:
https://github.com/electron/electron/issues/15112